packages/pg-packet-stream/dist/BufferReader.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
```

Comment:
pg-packet-stream’s `dist` seems to have been added unintentionally?

Reply:
ah, boo - thought I put an ignore in for that. will fix, thanks for the catch!

Reply:
i rebased that mistake out of the history on this branch.